infrastructure/deployment.json

```diff
@@ -485,7 +479,7 @@
                         },
                         {
                             "name": "AZURE_OPENAI_PREVIEW_API_VERSION",
```

Comment:
We can actually just remove this item altogether - let the code read the default instead of setting it at deployment time. Then the app will auto update with the new code container.